packages/flutter_tools/lib/src/commands/emulators.dart

```diff
@@ -16,13 +16,18 @@ class EmulatorsCommand extends FlutterCommand {
   EmulatorsCommand() {
     argParser.addOption('launch',
         help: 'The full or partial ID of the emulator to launch.');
+    argParser.addFlag('create',
+        help: 'Creates a new Android emulator.',
```

Comment:
We may want to make this a bit longer; we could say that we'll try and create a pixel device, and perhaps include a goo.gl shortened link for more info (until we have a short link to a flutter.io page).

Reply:
I added "based on a Pixel device" though not sure what more to add. Do we have somewhere reasonable to link to? The current links we have (in the help) go to pages on managing AVDs with the Android SDK which might not be totally appropriate here?